specs/latest/2.0/index.html

```diff
@@ -3508,8 +3508,8 @@ <h3><a name="ENABLED_ATTRIBUTE">Enabled Attribute</a></h3>
     <p>
```

Comment:
Remove this whole p and div. It's totally valid to draw without any enabled arrays about to active attribs.

Reply:
Updated the pull request to remove the whole section and also added a new test covering this. The test passes in Chrome and Firefox on Windows/NVIDIA at least.